server/graphql/v2/mutation/LegalDocumentsMutations.ts

```diff
@@ -124,4 +126,54 @@ export const legalDocumentsMutations = {
       return legalDocument;
     },
   },
+  editLegalDocumentStatus: {
```

Comment:
Introduced as an `editLegalDocumentStatus` mutation as we can anticipate that https://github.com/opencollective/opencollective/issues/6225 will follow a similar pattern.